src/nodes/interceptor/eth_sign_transaction.go

```diff
@@ -57,7 +53,14 @@ func (i *Interceptor) ethSignTransaction(ctx context.Context, msg *ethereum.Send
 	if msg.IsPrivate() {
```

Comment:
I would rather replace this code by a switch stament to make it more readable:
```
switch
  case msg.IsPrivate():
  case msg.IsLegacy():
  default:
    // DynamicFeeTxType

Reply:
done